tests/unit/test_miles_to_km_pytest.py

```diff
@@ -19,6 +19,6 @@ def test_miles_to_km_dict_type():
 
 def test_miles_to_km_list_type():
     with pytest.raises(TypeError):
-        mile_to_km([1,2,3])
+        miles_to_km([1,2,3])
```

Comment:
This is an intentional error. Please read article 
https://pytest-with-eric.com/comparisons/pytest-vs-unittest/#Testing-%E2%80%94-Pytest

Reply:
You can close this PR